client/mysqldump.c

```diff
@@ -5652,7 +5693,10 @@ static int start_transaction(MYSQL *mysql_con)
     if (!ignore_errors)
       exit(EX_MYSQLERR);
   }
-
+  if (has_session_variables_like(mysql_con, "rocksdb_skip_fill_cache"))
+    mysql_query_with_error_report(mysql_con, 0,
+                                  "/*!50601 SET SESSION"
```

Comment:
No need for !50601: the variable name is checked for, and SET SESSION itself has been here forever